lib/shared/utils.js

```diff
@@ -22,18 +22,20 @@ const errors = require('./errors');
 /**
  * @summary Minimum percentage value
  * @constant
- * @private
+ * @public
  * @type {Number}
  */
 const PERCENTAGE_MINIMUM = 0;
+exports.PERCENTAGE_MINIMUM = PERCENTAGE_MINIMUM;
```

Comment:
Haha, fair enough. Regarding the declaration, `exports.PERCENTAGE_MINIMUM = 0` should work (without the `const`).